host/libvirt_lxc_backend.go

```diff
@@ -663,12 +666,27 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
-		go log.Follow(3, initLog)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app_name"],
```

Comment:
want me to swap `JobID` and `AppName` so that `ProcID=<config.AppName>.<config.JobType>` and `AppName=<config.JobID>`?